static/index.html

```diff
@@ -51,26 +53,24 @@
         <main class="normalize" id="grapheneos">
             <div class="content hero">
 
-                <div>
+                <div class="hero-text">
                     <h1><a href="#grapheneos">GrapheneOS</a></h1>
-                    <p>The private and secure mobile operating system with Android app compatibility.
-                        Developed as a non-profit open source project.</p>
-                    <a class="button" href="/install/">Install GrapheneOS</a>
+                    <p>Privacy and security, crafted for your mobile device, usable by everyone.</p>
+                    <a id="install-button" class="button" href="/install/">Install Now</a>
+                    <p class="hero-little">
+                        Available for Google Pixel 4 and newer Google Pixel phones.
```

Comment:
Maybe add a link to https://en.wikipedia.org/wiki/Google_Pixel#Phones on `Google Pixel Phones`?

Reply:
Is this necessary? I don't believe a link to the wiki page is found anywhere else on the site, although I could be wrong.

Reply:
Done.